lib/google/ads/google_ads/deprecation.rb

```diff
@@ -0,0 +1,100 @@
+module Google
+  module Ads
+    module GoogleAds
+      class Deprecation
+        Error = Class.new(StandardError)
```

Comment:
Your commit message is "add tests" but I don't actually see any tests. Did you miss committing some?